testsuite/run_sets/testsuite.yml

```diff
@@ -30,8 +30,8 @@
 - features/core_centos_salt_ssh.feature
 - features/core_ubuntu_salt_ssh.feature
 # these features sync real channels (last core features)
-- features/core_srv_sync_channels.feature
 - features/core_srv_setup_wizard.feature
+- features/core_srv_sync_channels.feature
```

Comment:
Moving around core features is never guaranteed to work.

Did you test it did not introduce problems?